mu4/importexport/internal/braille/exportbraille.cpp

```diff
@@ -2481,6 +2488,7 @@ QString ExportBraille::brailleMarker(Marker* marker)
     case Marker::Type::FINE:
         return BRAILLE_FINE;
     case Marker::Type::TOCODA:
+    case Marker::Type::TOCODASYM:
```

Comment:
added recently, I guess it is OK to treat it the same as the 'normel' to COda, @andreituicu